src/bin/webextension/input.rs

```diff
@@ -0,0 +1,53 @@
+use byteorder::{NativeEndian, ReadBytesExt};
+use serde::Deserialize;
+use std::io::Read;
+use std::io::StdinLock;
+use std::{error, fmt};
+
+#[derive(Debug)]
+pub(crate) enum InputError {
+    IOError,
+    JSONError,
+}
```

Comment:
There's a few things that can be improved about the custom error you are introducing here. I'll give it a try later.

Reply:
I've pushed some changes and opened a pull request for it: https://github.com/dlalic/resume/pull/21

Reply:
Thanks.